.tx/config

```diff
@@ -0,0 +1,17 @@
+[main]
+host = https://www.transifex.com
+
+[o:web-scrobbler:p:web-scrobbler:r:messages-json--master]
+file_filter   = src/_locales/<lang>/messages.json
+source_file   = src/_locales/en/messages.json
+type          = CHROME
+minimum_perc  = 0
+resource_name = messages.json
+
+[o:web-scrobbler:p:web-scrobbler:r:privacy-md--master]
```

Comment:
I think it can probably stay, and we can rehandle it in the future

Reply:
Need to look into whats happening here, for some langs its pulling down in source lang :/

Reply:
I checked on tx when you gave me admin and at least for the german stuff I found that the translations are not on transifex, probably added directly to repo at some point. I think this might be the case across the board but I didn’t check extensively

Reply:
Fixed up - was missing some bits :/